csharp/src/truelayer-signing.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>0.1.0</Version>
+    <TargetFramework>net5.0</TargetFramework>
+    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
+    <Nullable>enable</Nullable>
+    <WarningsAsErrors>CS8600;CS8602;CS8603;CS8604;CS8618</WarningsAsErrors>
+    <Authors>Alex Butler</Authors>
```

Comment:
I think we should put `TrueLayer` here as we publish it from our Org 🤔

Reply:
Also maybe add:
```
<Company>TrueLayer</Company>
<Copyright>TrueLayer (c) $([System.DateTime]::Now.ToString(yyyy))</Copyright>
```
And the github info tags:
```
<PackageProjectUrl>https://github.com/truelayer/{repository}</PackageProjectUrl>
    <PackageReleaseNotes>See $(PackageProjectUrl)/releases for details.</PackageReleaseNotes>
    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
    <Repository>truelayer-signing</Repository>
    <RepositoryType>git</RepositoryType>
    <RepositoryUrl>$(PackageProjectUrl).git</RepositoryUrl>
```

Reply:
Thanks I left my name in authors, as from what I can see this is generally done. But I don't mind too much either way.